docs/tutorials/end_to_end_example.ipynb

```diff
@@ -26,7 +26,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    vis_cpu_sim = VisibilitySimulation(data_model=data_model, simulator=MatVis())
</pre><p>Should we change this (and following occurrences) to <code>matvis_sim</code>?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>